subsys/serialization/transport/rp_ll.c

```diff
@@ -0,0 +1,335 @@
+/*
```

Comment:
I consider this file not as long term solution. It will be replaced by generic module created by Low Level team.